shared/agent/src/providers/newrelic/anomalyDetectionDrillDown.ts

```
@@ -580,7 +581,12 @@ export class AnomalyDetectorDrillDown {
 	}
 
 	private runNrql<T>(nrql: string): Promise<T[]> {
-		return this.graphqlClient.runNrql(this._accountId, nrql, 400);
+		try {
+			return this.graphqlClient.runNrql(this._accountId, nrql, 400);
+		} catch (ex) {
+			Logger.error(ex);
+			return Promise.resolve([]);
+		}
```

The caller should choose to ignore errors or not - APIs should always return errors. If you do it here someone a year for now will have a very bad time figuring out why something silently fails. (logging an error and returning the error is a common pattern so the logging of the error won't necessarily make it stand out)

---

@dsellarsnr  I might take this out, because following up the chain, it seems there are plenty of try/catches around the topmost calls anyway. In those cases, I'm not entirely certain anymore this is what's causing the blank CS pane